CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added|Changed|Deprecated|Removed|Fixed|Security
 Nothing so far
 
+## 4.0.8 - 2019-08-13
+### Fixed
+- The `UrlMapperPass` will now use an optional `ordinal`.
```

Comment:
De gebruikelijke term hiervoor in Symfonyland is `priority`. Ik zou willen voorstellen om ook deze term hier te gebruiken zodat je daar niet over hoeft na te denken.

`The priority is optional and its value is a positive or negative integer that defaults to 0. The higher the number, the earlier that warmers are executed.`

```xml
<service id="security.access.simple_role_voter" class="Symfony\Component\Security\Core\Authorization\Voter\RoleVoter">
       <tag name="security.voter" priority="245" />
</service>
```

Reply:
Ik had dat bewust niet gedaan omdat ik heb bij 'priority' altijd lastig vindt of een laag getal nu voor of juist na een hoog getal komt.  Maar ik ben het wel met je eens, zal het aanpassen.

Heb je nog eens wens over de default waarde, aangezien je hier 245 hebt staan?

Reply:
Ja, dat vind ik ook verwarrend, maar wel consistent verwarrend :upside_down_face: